messages.md

```diff
@@ -95,6 +99,9 @@ These JSON hashes included in the table above are broken out into their own tabl
 |statistics.unsubscribed	|integer    |A read-only computed property representing the total number of messages where the person receiving the message unsubscribed.
 |statistics.bounced	|integer    |A read-only computed property representing the total number of messages that bounced or were otherwise undelivered.
 |statistics.spam_reports	|integer    |A read-only computed property representing the total number of messages marked as spam by people receiving them.
+|statistics.delivered       |integer    |A read-only computed property representing the SMS messages that were delivered to the recipients. (Applicable to SMS messages)

```

Comment:
Thanks for your review Jason. In SMS campaigns, 'sent' represents the number of texts that were sent from the originating system. A given text message is classified as 'delivered' when the telecom carrier confirms the delivery of message to the recipient mobile. A good delivery rate is critical to success of SMS campaigns, so we feel 'delivered' would add value here.